index.html

```diff
@@ -4,7 +4,194 @@
+        <li>
+          <a class="hamburguer-link" href="about.html"><h3 class="h3">About</h3></a>
+        </li>
+      </ul>
+    </div>
+
+    <main>
+      <section id="heading" class="heading">
+        <h2>"Hello Reggae Lover"</h2>
+        <h1>
+          Protoje Reggae <br />
+          Munich Concert <br />
+          2023
+        </h1>
+        <p class="heading">
+          Munich is a vibrant city with a rich cultural history, and reggae music has played a significant role in shaping its
+          music scene.
+        </p>
+        <h3>2022.03.27(FRI) and 28(SAT)</h3>
+        <h4>@Olympiahalle Arena, Munich - Deutschland</h4>
+      </section>
+
+      <section id="mainProgram" class="main_program">
+        <h2 class="main_program">Main Program</h2>
+        <hr class="hr1" />
+        <ul class="main_program">
+          <li class="main_program">
+            <div class="main_program">
+              <img src="./assets/images/writing.png" alt="Autograph Session" />
+            </div>
+            <h2>Autograph Session</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program">
+            <div class="main_program">
+              <img src="./assets/images/video-camera.png" alt="New Video Recording" />
+            </div>
+            <h2>Video Recording</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program">
+            <div class="main_program">
+              <img src="./assets/images/ear.png" alt="Test Sound" />
+            </div>
+            <h2>Test Sound</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program col-sm-12 col-md-6 col-lg-2">
+            <div class="main_program">
+              <img src="./assets/images/concert.png" alt="Concert" />
+            </div>
+            <h2>Concert</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+
+          <li class="main_program col-sm-12 col-md-6 col-lg-2">
+            <div class="main_program">
+              <img src="./assets/images/reggae.png" alt="After Party" />
+            </div>
+            <h2>After Party</h2>
+            <p class="main_program">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </li>
+        </ul>
+        <div class="mp_div_mob">
+          <button type="button">Buy a Ticket Now!</button>
+        </div>
+        <div class="mp_div_dsk">
+          <a class="mp_div_dsk" href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+
+      <section id="featuredPerformers" class="featured_performers"></section>
+    </main>
+
+    <footer>
+      <div class="footer1">
+        <h2 class="footer1">Partner</h2>
+        <hr class="hr1" />
```

Comment:
- [ ] Kindly make sure this title is underlined as per the design provided in the requirements. :recycle: 

  - Your design:
![Ypartner](https://user-images.githubusercontent.com/97834160/211686262-aeddc2f2-eff1-4bc6-8c21-ce2cb11b1cc3.png)


  - Cindy's design:
![Bpartner](https://user-images.githubusercontent.com/97834160/211686293-fac78c92-19ca-4716-a84b-c74401600430.png)